src/reactions/MessageReactionList.js

```diff
@@ -16,6 +23,13 @@ import tabsOptions from '../styles/tabs';
 import Emoji from '../emoji/Emoji';
 import { objectFromEntries } from '../jsBackport';
 
+const emojiTypeReactionType = (reactionType: ReactionType): EmojiType => {
```

Comment:
Needs a word in the middle to relate these two things :slightly_smiling_face: Try `emojiTypeFromReactionType`.